src/compat/fasttext/indexer.rs

```diff
@@ -44,6 +44,10 @@ impl Indexer for FastTextIndexer {
     fn index_ngram(&self, ngram: &StrWithCharLen) -> u64 {
         u64::from(fasttext_hash(ngram.as_str()) % self.buckets)
     }
+
+    fn upper_bound(&self) -> u64 {
+        u64::from(self.buckets) + 1
```

Comment:
`u64::from(self.buckets)` is correct, since the buckets are numbered from 0, the upper bound is `u64::from(self.buckets)`, the largest possible index `u64::from(self.buckets) - 1`